### DISCUSSION TITLE
Welcome to motion Discussions!

### DISCUSSION THREAD

**Discussion by latortuga** [unabridged/motion#69] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jhirbour** [comment]
I'm currently a Backend Engineer going some "light UI work".  This gem looks to me like Rails-USJ COULD have been (easy to use etc). I haven't actively starting using this yet, but it looks awesome! Keep up the good work!
